app/src/main/java/fr/free/nrw/commons/AboutActivity.java

```diff
@@ -114,6 +116,10 @@ public void launchCredits(View view) {
         Utils.handleWebUrl(this, Uri.parse(Urls.CREDITS_URL));
     }
 
+    public void launchUserGuide(View view) {
```

Comment:
You can add test for this method

Reply:
done sir

Reply:
Thanks for adding the Instrumentation test, you can also add a unit test for this method.